tests/internal/tests/gw/test.go

```diff
@@ -0,0 +1,71 @@
+package gw
+
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	vegeta "github.com/tsenart/vegeta/v12/lib"
+	"github.com/tsenart/vegeta/v12/lib/prom"
+)
+
+const (
+	frequency = 1000
+	duration  = 5 * time.Minute
+)
+
+func NewLoadTest(pm *prom.Metrics, url string) *LoadTest {
+	return &LoadTest{
+		pm:  pm,
+		url: url,
+	}
+}
+
+type LoadTest struct {
+	pm  *prom.Metrics
```

Comment:
again, too short variable names are confusing

Reply:
@Hukyl 

> The general rule of thumb is that the length of a name should be proportional to the size of its scope and inversely proportional to the number of times that it is used within that scope. A variable created at file scope may require multiple words, whereas a variable scoped to a single inner block may be a single word or even just a character or two, to keep the code clear and avoid extraneous information.
> 
> Here is a rough baseline. These numeric guidelines are not strict rules. Apply judgement based on context, [clarity](https://google.github.io/styleguide/go/guide#clarity), and [concision](https://google.github.io/styleguide/go/guide#concision).
> 
> A small scope is one in which one or two small operations are performed, say 1-7 lines.
> A medium scope is a few small or one large operation, say 8-15 lines.
> A large scope is one or a few large operations, say 15-25 lines.
> A very large scope is anything that spans more than a page (say, more than 25 lines).
> A name that might be perfectly clear (e.g., c for a counter) within a small scope could be insufficient in a larger scope and would require clarification to remind the reader of its purpose further along in the code. A scope in which there are many variables, or variables that represent similar values or concepts, may necessitate longer variable names than the scope suggests.


pm variable is used in two places only in this file, so, I think it could be classified as small-medium, so I don't see point in giving it more descriptive name, according to [Google's style guide](https://google.github.io/styleguide/go/decisions#naming)

Reply:
@Hukyl hm, but if it's medium-large then I can increase the length a bit. I need to think about it

Reply:
@Hukyl I think I may extend it to one word. That would be perfectly enough for internal variables used in one file. Thank you for pointing out